File: src/main/java/com/wikia/tibia/repositories/MountRepository.java
package com.wikia.tibia.repositories;

import com.wikia.tibia.gateways.MountGateway;
import com.wikia.tibia.objects.Mount;

public class MountRepository extends WikiObjectRepository {

    public MountRepository() {
        super(Mount.class, new MountGateway());
    }
}

File: src/main/kotlin/com/wikia/tibia/repositories/MountRepository.kt
package com.wikia.tibia.repositories

import com.wikia.tibia.enums.Contract
import com.wikia.tibia.gateways.WikiObjectGateway
import com.wikia.tibia.objects.Mount

class MountRepository : WikiObjectRepository<Mount>(
        wikiObjectClass = Mount::class.java,
        wikiObjectGateway = WikiObjectGateway(Contract.MOUNTS)
)